static/test/javascripts/spec/common/commercial/membership-engagement-banner.spec.js

```diff
@@ -69,13 +72,15 @@ define([
 
         function expectMessageNotToBeShown(done) {
             membershipMessages.init().then(function () {
+                mediator.emit('modules:onwards:breaking-news:ready', false);
                 var message = document.querySelector('.js-site-message');
                 expect(message).toBeNull();
             }).then(done);
         }
 
         function expectMessageNotToBeVisible(done) {
             membershipMessages.init().then(function () {
+                mediator.emit('modules:onwards:breaking-news:ready', false);
```

Comment:
Yeah that's a good idea. It would be sensible to have tests to check the following two things (just making notes for myself here):

- The banner doesn't show when breaking news shows.
- The banner does show when breaking news doesn't (all other conditions being ok).

These tests should perhaps run first, because if something goes wrong and these conditions don't hold true then some of the other tests may fail/pass for the wrong reason.